CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Welcome to CONTRAST RATIO APP's contributing guide
+
+:+1::tada: First off, thank you for investing your time in contributing to our project! :tada::+1:
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+Use the table of contents icon on the top left corner of this document to get to a specific section of this guide quickly.
+
+## Code of Conduct
+
+Please, read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+## New contributor guide
+
+To get an overview of the project, please read the [README](./README.md).
+
+## Getting started
+
+### Issues
+
+#### Create a new issue
```

Comment:
Just checking out how `####` type of headings looks on Github:

#### testing heading

It doesn't look like a heading. I know it's just little h4, but still not sure.
Maybe we need to give this heading little more accent? One option woulbe to italicicze.

#### *Italicized testing heading*

Reply:
I agree. I will go ahead and italicize these headings.